Sources/AsyncHTTPClient/Docs.docc/index.md

```diff
@@ -31,14 +31,13 @@ and  `AsyncHTTPClient` dependency to your target:
 
 The code snippet below illustrates how to make a simple GET request to a remote server.
 
-Please note that the example will spawn a new `EventLoopGroup` which will _create fresh threads_ which is a very costly operation. In a real-world application that uses [SwiftNIO](https://github.com/apple/swift-nio) for other parts of your application (for example a web server), please prefer `eventLoopGroupProvider: .shared(myExistingEventLoopGroup)` to share the `EventLoopGroup` used by AsyncHTTPClient with other parts of your application.
-
-If your application does not use SwiftNIO yet, it is acceptable to use `eventLoopGroupProvider: .createNew` but please make sure to share the returned `HTTPClient` instance throughout your whole application. Do not create a large number of `HTTPClient` instances with `eventLoopGroupProvider: .createNew`, this is very wasteful and might exhaust the resources of your program.
-
 ```swift
 import AsyncHTTPClient
 
-let httpClient = HTTPClient(eventLoopGroupProvider: .createNew)
+let httpClient = HTTPClient(eventLoopGroupProvider: .singleton)
```

Comment:
Is there any reason these examples don't use the zero-element initializer?

Reply:
@Lukasa Yes, that's only coming in #705 . #705 does also rewrite all these examples to just use `HTTPClient.shared`. But I split this work into two parts (maybe misguidedly):

1. Introduce the `.singleton` `EventLoopGroupProvider` --> this PR
2. Introduce `HTTPClient.shared`, a completely shared singleton HTTPClient that doesn't need shutdown --> #705